build/registry/resyncRsyncRegistry

```diff
@@ -11,10 +11,9 @@ cat > /etc/rsyncd.conf << RD
 uid = root
 gid = root
 use chroot = yes
-max connections = 4
+max connections = 50
```

Comment:
Where's this used.

Reply:
we need to allow more than 4 downloads at a time on the prod registry